src/help_parser/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "help_parser"
+version = "0.0.0"
+edition = "2021"
+publish = false
```

Comment:
We'll need to publish it right? Because `uucore` depends on it.

Reply:
Yes, though only `uucore_procs` depends on it.